ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -167,48 +168,64 @@ public struct AvatarGroup: View {
-            }
-            if overflowCount > 0 {
-                createOverflow(count: overflowCount)
+        let groupHeight: CGFloat = imageSize + (ringOffset * 2)
+
+        @ViewBuilder
+        var avatarGroupContent: some View {
+            GeometryReader { geo in
+                HStack(spacing: 0) {
+                    ForEach(enumeratedAvatars.prefix(maxDisplayedAvatars), id: \.1) { index, avatar in
+                        // If the avatar is part of Stack style and is not the last avatar in the sequence, create a cutout
+                        let avatarView = avatarViews[index]
+                        let needsCutout = tokens.style == .stack && (overflowCount > 0 || index + 1 < maxDisplayedAvatars)
+                        let avatarSize: CGFloat = avatarView.state.totalSize()
+                        let nextAvatarSize: CGFloat = needsCutout ? avatarViews[index + 1].state.totalSize() : 0
+                        let isLastDisplayed = index == maxDisplayedAvatars - 1
+
+                        let currentAvatarHasRing = avatar.isRingVisible
+                        let nextAvatarHasRing = index + 1 < maxDisplayedAvatars ? avatars[index + 1].isRingVisible : false
+                        let avatarSizeDifference = avatarSize - nextAvatarSize
+                        let sizeDiff = !isLastDisplayed ? (currentAvatarHasRing ? avatarSizeDifference : avatarSizeDifference - ringGapOffset) :
+                            currentAvatarHasRing ? (avatarSize - ringGapOffset) - imageSize : (avatarSize - (ringGapOffset * 2)) - imageSize
+                        let x = avatarSize + tokens.interspace - ringGapOffset
+
+                        let ringPaddingInterspace = nextAvatarHasRing ? interspace - (ringOffset + ringOuterGap) : interspace - ringOffset
+                        let noRingPaddingInterspace = nextAvatarHasRing ? interspace - ringOuterGap : interspace
+                        let rtlRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOuterGap : -x + ringOffset)
+                        let rtlNoRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOffset - ringOuterGap : -x)
+                        let stackPadding = (currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace)
+
+                        let xPosition = currentAvatarHasRing ? x - ringOuterGap - ringOuterGap : x - ringOuterGap
+                        let xPositionRTL = currentAvatarHasRing ? rtlRingPaddingInterspace : rtlNoRingPaddingInterspace
+                        let xOrigin = Locale.current.isRightToLeftLayoutDirection() ? xPositionRTL : xPosition
+                        let yOrigin = sizeDiff / 2
+                        let cutoutSize = isLastDisplayed ? (ringOuterGap * 2) + imageSize : nextAvatarSize
+
+                        avatarView
+                            .modifyIf(needsCutout, { view in
+                                view.mask(Avatar.AvatarCutout(
```

Comment:
You need to incorporate #703 here.
This state of the code still reproes the iOS 13.0/13.1 issue:

![Screen Shot 2021-09-10 at 1 46 44 PM](https://user-images.githubusercontent.com/68076145/132915759-3241637d-0b73-48d1-a962-d8996008cfb8.png)